src/Campaigns/Blocks/CampaignDonorsBlock/render.php

```diff
@@ -0,0 +1,261 @@
+<?php
+
+namespace Give\Campaigns\Blocks\CampaignDonorsBlock;
+
+use Carbon\Carbon;
+use Give\Campaigns\CampaignDonationQuery;
+use Give\Campaigns\Models\Campaign;
+use Give\Campaigns\Repositories\CampaignRepository;
+use Give\DonationForms\Blocks\DonationFormBlock\Controllers\BlockRenderController;
+use Give\Donations\ValueObjects\DonationMetaKeys;
+use Give\Framework\Support\ValueObjects\Money;
+
+/**
+ * @unreleased
+ */
+if ( ! class_exists(CampaignDonorsBlockRenderer::class)) {
+    class CampaignDonorsBlockRenderer
```

Comment:
Shouldn't this be in it's own namespace file so you don't have to check if class exists?

Reply:
I had to add that check after receiving a fatal error on the editor due to the class being registered multiple times.

Reply:
Refactored on 9392c1085bb153535a92e0fae8d0c3ce9b478e47